### PR TITLE
Enhance @addons endpoint to return a list of upgradeable addons

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by jensens** [comment]
relates to issue #1319

File: src/plone/restapi/services/addons/get.py [hunk 1]
```diff
@@ -31,6 +32,17 @@ def reply(self):
...
             addons_data.append(self.addons.serializeAddon(addon))
+
+        if len(self.query) > 0 and len(self.params) == 0:
+            upgradeables = self.query.get("upgradeable", "")
+            if upgradeables:
+                addons_data = [
+                    addon
+                    for addon in addons_data
+                    if addon.get("upgrade_info", False)
+                    and addon["upgrade_info"].get("available", False) is True
```
Comment (jensens) [inline src/plone/restapi/services/addons/get.py:43]: It's already Bool.
```suggestion
                    and addon["upgrade_info"].get("available", False)
```

**Review by jensens — APPROVED** [review]
One odd expression, otherwise fine.

**Review by tisto — APPROVED** [review]

File: src/plone/restapi/services/addons/get.py (hunk 1 above)
Comment (sneridagh) [reply src/plone/restapi/services/addons/get.py]: Done!

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@tisto please, merge and release whenever you can.

**Review by tisto — APPROVED** [review]

**Comment by Rudd-O** [comment]
Wow, fellas, this is really coming together.

**Pull request merged**